node/src/accountant/payment_adjuster/miscellaneous/account_stages_conversions.rs

```diff
@@ -0,0 +1,156 @@
+    use crate::accountant::payment_adjuster::miscellaneous::data_structures::{
+        AdjustedAccountBeforeFinalization, UnconfirmedAdjustment, WeightedPayable,
+    };
+    use crate::accountant::test_utils::{
+        make_non_guaranteed_qualified_payable, make_payable_account,
+    };
+    use crate::accountant::AnalyzedPayableAccount;
+
+    #[test]
+    fn conversion_between_non_finalized_account_and_payable_account_is_implemented() {
+        let mut original_payable_account = make_payable_account(123);
+        original_payable_account.balance_wei = 200_000_000;
+        let non_finalized_account = AdjustedAccountBeforeFinalization::new(
+            original_payable_account.clone(),
+            666777,
+            123_456_789,
+        );
+
+        let result = PayableAccount::from(non_finalized_account);
+
+        original_payable_account.balance_wei = 123_456_789;
+        assert_eq!(result, original_payable_account)
+    }
+
+    fn prepare_weighted_account(payable_account: PayableAccount) -> WeightedPayable {
+        let garbage_disqualification_limit = 333_333_333;
+        let garbage_weight = 777_777_777;
+        let mut analyzed_account = AnalyzedPayableAccount::new(
+            make_non_guaranteed_qualified_payable(111),
+            garbage_disqualification_limit,
+        );
+        analyzed_account.qualified_as.bare_account = payable_account;
+        WeightedPayable::new(analyzed_account, garbage_weight)
+    }
+
+    #[test]
+    fn conversation_between_weighted_payable_and_standard_payable_account() {
+        let original_payable_account = make_payable_account(345);
+        let weighted_account = prepare_weighted_account(original_payable_account.clone());
+
+        let result = PayableAccount::from(weighted_account);
+
+        assert_eq!(result, original_payable_account)
+    }
+
+    #[test]
+    fn conversion_between_weighted_payable_and_non_finalized_account() {
+        let original_payable_account = make_payable_account(123);
+        let mut weighted_account = prepare_weighted_account(original_payable_account.clone());
+        weighted_account
+            .analyzed_account
+            .disqualification_limit_minor = 200_000_000;
+        weighted_account.weight = 78910;
+
+        let result = AdjustedAccountBeforeFinalization::from(weighted_account);
+
+        let expected_result =
+            AdjustedAccountBeforeFinalization::new(original_payable_account, 78910, 200_000_000);
+        assert_eq!(result, expected_result)
+    }
+
+    #[test]
+    fn conversion_between_unconfirmed_adjustment_and_non_finalized_account() {
+        let mut original_payable_account = make_payable_account(123);
+        original_payable_account.balance_wei = 200_000_000;
+        let mut weighted_account = prepare_weighted_account(original_payable_account.clone());
+        weighted_account.weight = 321654;
+        let unconfirmed_adjustment = UnconfirmedAdjustment::new(weighted_account, 111_222_333);
```

Comment:
It might look better if that 111_222_333 was assigned to a variable with a self-explaining name.